edgelet/kube-client/src/config.rs

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::env;
+use std::fs;
+use std::path::Path;
+
+use failure::Fail;
+use log::info;
+use native_tls::{Certificate, Identity, TlsConnector};
+use openssl::pkcs12::Pkcs12;
```

Comment:
Ok, so when I wrote this I didn't have to worry about Windows. But now, this stuff will fail to build on Windows because.. well.. we need to have OpenSSL installed which though we do (because our HSM C lib needs it), it isn't installed quite the way the Rust OpenSSL crate likes it.

Reply:
So, I can change the build script to get this to run....  Is that what we want to do?

Reply:
That might be the best short-term solution. From what I remember, we need to setup an environment variable named `OPENSSL_DIR` that points to where it is installed.

Long term though, I want to see if we can use a pure-rust solution for what OpenSSL is being used here. But that can come later.

Reply:
You can use winapi for Windows: https://github.com/Arnavion/k8s-openapi-codegen/blob/master/k8s-openapi-tests/src/client_winapi.rs + https://github.com/Arnavion/k8s-openapi-codegen/blob/master/k8s-openapi-tests/src/lib.rs#L110-L117

Reply:
Thank you, I also added this as a task.